### PR TITLE
add category table

### PR DESCRIPTION
**Pull request by IrinaLokhvytska**
1. Create category table
2. Edit mapper.py

### CODE REVIEW

File: database/models.py [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 
 
+class Category(Base):
+    __tablename__ = 'category'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    timestamp = Column('timestamp', DateTime, default=datetime.utcnow())
+    title = Column('title', String(255), nullable=False)
+    amount = Column('amount', Float, nullable=True)
+    start_date = Column('start_date', Date, nullable=True)
```
Comment (evgeniyDorogoy) [inline database/models.py]: As I remember we decide to set up time ranges for limits - day (1), week (7), month (30), year (365). Does that make sense?

Comment (IrinaLokhvytska) [reply database/models.py]: We can calculate end_date : `start_date + limit`

Comment (evgeniyDorogoy) [reply database/models.py]: What do you mean for `limit`? Does it `date` object? I guess that `limit` is absolute value of expences, e.g `10000 UAH`.

File: processor/mapper.py [hunk 2]
```diff
@@ -26,3 +26,47 @@ def insert_transactions(transactions, engine) -> None:
...
+
+def add_category_2_bd(category, engine: str) -> int:
+    '''
+        Inserts prepared data to the db table.
+        All data must correspond with Category model.
+
+        :param category: the name of the category.
+        :param engine: connected database engine.
+    '''
+    session = Session(bind=engine)
```
Comment (evgeniyDorogoy) [inline processor/mapper.py]: You shouldn't create new session for this. You can just put `Category` object into `Transaction` and that it.

```
category = Category(title='travel')
transaction = Transaction(category=category)
```

File: database/models.py (hunk 1 above)
Comment (IrinaLokhvytska) [reply database/models.py]: `Amount` is absolute value of expenses. Our range contains start_date and end_date.

File: processor/mapper.py (hunk 2 above)
Comment (IrinaLokhvytska) [reply processor/mapper.py]: `can't adapt type 'Category' [SQL: 'INSERT INTO transaction (timestamp, date, account, category, amount, currency, converted_amount, converted_currency, description, is_debet) VALUES (%(timestamp)s, %(date)s, %(account)s, %(category)s, %(amount)s, %(currency)s, %(converted_amount)s, %(converted_currency)s, %(description)s, %(is_debet)s) RETURNING transaction.id'] [parameters: {'timestamp': datetime.datetime(2018, 4, 23, 9, 34, 3, 918952), 'date': '05/04/2018', 'account': 'Счети4', 'category': <db.transaction.Category object at 0x7f74d342cac8>, 'amount': -258.0, 'currency': 'UAH', 'converted_amount': -258.0, 'converted_currency': 'UAH', 'description': nan, 'is_debet': False}]
`

**Pull request merged**